apps/asap-server/test/fixtures/teams.fixtures.ts

```diff
@@ -418,6 +422,7 @@ export const graphQlTeamResponse: { data: ResponseFetchTeam } = {
             flatData: {
               link: null,
               publishDate: null,
+              labCatalogNumber: 'http://example.com',
```

Comment:
i see you've changed the defaulting to undefined but you didn't change the tests - this is how i realised you never really tested the scenario when the `labCatalogNumber` is missing. I'd remove it from one of the ROs here and then update the expectation too just to cover that scenario